roles/docker/templates/custom.conf.j2

```diff
@@ -1,5 +0,0 @@
-# {{ ansible_managed }}
-
-[Unit]
-Requires=iptables.service
-After=iptables.service
```

Comment:
It looks like the firewalld unit file sets Before=network.target, which ensures the correct ordering. I suspect this workaround would still be needed for the case where os_firewall_use_firewalld is explicitly set to False.